arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to list log profiles",
+            "schema": {
+              "$ref": "#/definitions/LogProfileCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/{resourceUri}/providers/microsoft.insights/diagnosticSettings/service": {
+      "delete": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Delete",
+        "description": "Deletes the diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "name": "resourceUri",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier of the configuration."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-07-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to delete a service diagnostic setting"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Get",
+        "description": "Gets the active diagnostic settings. To get the diagnostic settings being applied, use GetStatus.",
+        "parameters": [
+          {
+            "name": "resourceUri",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier of the configuration."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-07-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get one service diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/ServiceDiagnosticSettingsResource"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Put",
+        "description": "Create or update new diagnostic settings for the specified resource. This operation is long running. Use GetStatus to check the status of this operation.",
```

Comment:
Please update the description. You no longer need the getStatus operation for polling the long running operation status. Tagging this with long running operation extension should be enough.